core/modules/replica/RebalanceJob.h

```diff
@@ -83,9 +83,9 @@ struct RebalanceJobResult {
 
     // Parameters of the planner
 
-    size_t totalWorkers    {0};     // not counting workers which failed to report chunks
-    size_t totalGoodChunks {0};     // good chunks reported by the precursor job
-    size_t avgChunks       {0};     // per worker average
+    size_t totalWorkers    = 0;     /// not counting workers which failed to report chunks
+    size_t totalGoodChunks = 0;     /// good chunks reported by the precursor job
+    size_t avgChunks       = 0;     /// per worker average
```

Comment:
For my education; why change from {} to =?

Reply:
According to the following section of the LSST C++ Style Guide:
"... Non-aggregate PODs should be constructed using an assignment-like syntax (note that this does not actually invoke the assignment operator):
```
double a = 4;           // good
double a{4};            // discouraged to maintain consistency
double a = {4};         // discouraged to maintain consistency
```
See:
https://developer.lsst.io/cpp/style.html#a-constructor-calls-should-use-c-98-syntax-when-possible

Reply:
That vague reason "...discouraged to maintain consistency..." is referring to consistency with Python 😆